src/interceptors/form-data.ts

```diff
@@ -0,0 +1,38 @@
+import { InternalAxiosRequestConfig, RawAxiosRequestHeaders } from 'axios';
+import { forOwn, some, values } from 'lodash';
```

Comment:
This is the one place this library uses lodash. Could you please refactor the code to avoid it and get-rid the dependency?